htdocs/accountancy/customer/index.php

```diff
@@ -196,11 +196,10 @@
 dol_syslog("htdocs/accountancy/customer/index.php sql=" . $sql, LOG_DEBUG);
 $resql = $db->query($sql);
 if ($resql) {
-	$i = 0;
 	$num = $db->num_rows($resql);
-	
-	while ( $i < $num ) {
-		$row = $db->fetch_row($resql);
+
+	while ( $row = $db->fetch_row($resql)) {
```

Comment:
Note that using fetch_row in future must be abandonned and replace with fetch_object